cola-tests/src/test/java/com/github/bmsantos/core/cola/instrument/ColaTransformerTest.java

```diff
@@ -0,0 +1,49 @@
+package com.github.bmsantos.core.cola.instrument;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+import static test.utils.TestUtils.loadClassBytes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import test.utils.StoriesFieldClass;
+
+public class ColaTransformerTest {
+
+    private ColaTransformer uut;
+
+    @Before
+    public void setUp() {
+        uut = new ColaTransformer();
+    }
+
+    @Test
+    public void shouldTransformClass() throws Exception {
+        // Given
+        final Class<?> clazz = StoriesFieldClass.class;
+        final byte[] original = loadClassBytes(clazz);
+
+        // When
+        final byte[] result = uut.transform(ColaTransformer.class.getClassLoader(), clazz.getName(), clazz, null,
+            original);
+
+        // Then
+        assertThat(result, not(equalTo(original)));
```

Comment:
No need. The other tests for the other classes already take care of it. In addition it is quite hard to test for bytecode differences.